### ISSUE TITLE
Request to set as waiting sets as completed

### ISSUE THREAD

**Issue by mjgiarlo** [sul-dlss/workflow-server-rails#89]
I had an errored state (which I manually set in the console after encountering #88) in the `submit-marc` step of the `etdSubmitWF` workflow. In argo-stage, I clicked the "set to waiting" button so I could re-run the step, and instead the step was set to completed. That button looks like this:

```html
<form action="https://argo-stage.stanford.edu/items/druid:pk473sh2989/workflows/etdSubmitWF" accept-charset="UTF-8" method="post">
  <input name="utf8" type="hidden" value="✓">
  <input type="hidden" name="authenticity_token" value="F7fKlrFmqYdT4zzZJIx339xAvmM5CpFWYdToV6e74870m1tn8rzu7CZxivIT8oGQfSaFCznt/bD4NF+DJBY5Mg==">
  <input type="hidden" name="process" id="process" value="submit-marc">
  <input type="hidden" name="status" id="status" value="waiting">
  <button name="button" type="submit">set to waiting</button>
</form>
```

**Comment by mjgiarlo** [comment]
I suspect https://github.com/sul-dlss/workflow-server-rails/pull/91 will fix this issue as well.

**Comment by mjgiarlo** [comment]
Fixed by #91. Just confirmed in argo-stage.
